### DISCUSSION TITLE
unable to connect to instance via web

### DISCUSSION THREAD

**Discussion by munagalasandeep99** [learndevopsonline/learndevopsonline#1884] [Q&A]
i have created a ec2 instance for my frontend microservice but iam unable to acces it from the web
![Screenshot from 2024-04-17 20-32-49](https://github.com/learndevopsonline/learndevopsonline/assets/129391713/14c42d05-ab24-4448-b625-94640e651e72)
i have enabled the security group inbounds to allow all
![Screenshot from 2024-04-17 20-38-35](https://github.com/learndevopsonline/learndevopsonline/assets/129391713/2802f570-9a4c-4768-91e8-3759631ca16a)

and still iam getting this
![Screenshot from 2024-04-17 20-37-55](https://github.com/learndevopsonline/learndevopsonline/assets/129391713/5a8651cd-ef4f-4d6a-bef3-d8ed7c390869)

this is my instance
![Screenshot from 2024-04-17 20-39-25](https://github.com/learndevopsonline/learndevopsonline/assets/129391713/00a06280-641a-4977-a283-ae0c931de778)

**Comment by chaithanya1812** [comment]
Hi @munagalasandeep99,
### Sorry for late reply,
## Isuue:- frontend-(nginx service) is not running...
